boilerplate/web_modules/layouts/Homepage/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, PropTypes } from "react"
+import enhanceCollection from "phenomic/lib/enhance-collection"
+
+import Page from "../Page"
+import PagesList from "../../PagesList"
+
+const numberOfLatestPosts = 6
+
+export default class Homepage extends Component {
+  static contextTypes = {
+    collection: PropTypes.array.isRequired,
+  }
+
+  render() {
+    const latestPosts = enhanceCollection(this.context.collection, {
+      filter: { layout: "Post" },
+      sort: "date",
+      reverse: true,
+    })
+    .slice(0, numberOfLatestPosts)
```

Comment:
enhanceCollection has a limit option for this job.

Maybe we can extend it to support pagination